elasticsearch.md

```diff
@@ -241,7 +241,7 @@ Creating a dedicated, lightweight ES node where to install ReadonlyREST:
 
 #### An exception
 
-**⚠️IMPORTANT** when `filter` or `fields` rules are used, it's required to install ReadonlyREST plugin in all the data nodes. This happens because these rules are implemented at Lucene level.
+**⚠️IMPORTANT** when `fields` rule with `es_with_lucene` FLS engine is used, it's required to install ReadonlyREST plugin in all the data nodes. This happens because this rule is partly implemented at Lucene level.
```

Comment:
Yeah it would be cool to have a separate page for this, and a TL;DR about it that does not mention the "engine" stuff because the average Joe is not gonna need it.